recipes/opentelemetry-sdk-extension-prometheus-multiprocess/recipe.yaml

```diff
@@ -0,0 +1,44 @@
+schema_version: 1
+
+context:
+  name: opentelemetry-sdk-extension-prometheus-multiprocess
+  version: 1.2.3
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/opentelemetry_sdk_extension_prometheus_multiprocess-${{ version }}.tar.gz
+  sha256: 53b3f9f10fee6acd7189c868324669156ca2324ad9c652b6b2418571413d60ca
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python
+    - hatchling
+    - hatch-fancy-pypi-readme
+    - pip
+  run:
+    - python
+    - opentelemetry-sdk >=1.30.0.dev0
+    - requests
+    - opentelemetry-exporter-prometheus
+
+tests:
+  - python:
+      imports:
+        - opentelemetry_sdk_extension_prometheus_multiprocess
+      pip_check: true
+
+about:
+  license: Unlicense
+  license_file: PLEASE_ADD_LICENSE_FILE
```

Comment:
homepage: https://github.com/ZeitOnline/opentelemetry-prometheus-multiprocess

License is tricky since upstream doesn't specify any kind of license anywhere. Have to work with upstream to actually provide a license somewhere.